src/os/signal/signal.go

```diff
@@ -1,14 +0,0 @@
-// Copyright 2012 The Go Authors. All rights reserved.
-// Use of this source code is governed by a BSD-style
-// license that can be found in the LICENSE file.
-
-package signal
-
-import (
-	"os"
-)
-
-// Just stubbing the functions for now since signal handling is not yet implemented in tinygo
-func Reset(sig ...os.Signal)                      {}
-func Ignore(sig ...os.Signal)                     {}
```

Comment:
What is the reason you want to remove these functions? They are widely used and we need an API to interact with the runtime. I proposed a fairly basic implementation for that in #4384, maybe we can do something similar here?

```suggestion
func Ignore(sig ...os.Signal) {
	for _, s := range sig {
		signal_ignore(uint32(signum(s)))
	}
}
```

Reply:
Instead of using our own implementation, we use the one in the Go standard library. That one is already well tested so as long as the runtime side is implemented correctly, the signal package should also be correct.

By removing the package entirely, loader/goroot.go will pick the one from the standard library.